sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -113,27 +117,24 @@ def send_events(self, events, **kwargs):
         if not isinstance(events, list):
             events = cast(ListEventType, [events])
 
-        if all(isinstance(e, CloudEvent) for e in events) or all(_is_cloud_event(e) for e in events):
+        if isinstance(events[0], CloudEvent) or _is_cloud_event(events[0]):
             try:
                 events = [cast(CloudEvent, e)._to_generated(**kwargs) for e in events] # pylint: disable=protected-access
             except AttributeError:
                 pass # means it's a dictionary
             kwargs.setdefault("content_type", "application/cloudevents-batch+json; charset=utf-8")
```

Comment:
non-blocking: we could consider constifying those strings if we use them across the sdk

Reply:
hmm, we can, but this is the only method where we use it

Reply:
if there're only one or two places, I'm good with not constifying the string :)